packages/deepbook/sources/balance_manager.move

```diff
@@ -8,12 +8,7 @@
 /// withdrawals, and addition / removals of traders.
 module deepbook::balance_manager {
     // === Imports ===
-    use sui::{
-        bag::{Self, Bag},
-        balance::{Self, Balance},
-        coin::Coin,
-        vec_set::{Self, VecSet},
-    };
+    use sui::{bag::{Self, Bag}, balance::{Self, Balance}, coin::Coin, vec_set::{Self, VecSet}};
```

Comment:
Which format should we follow? 2nd level formatting in book.move is separated into different lines

Reply:
It's not "we follow", we just format. If it fits onto a single line, formatter will try that.